lib/tasks/account.rake

```diff
@@ -25,13 +25,18 @@ namespace :"account" do
     end
   end
 
-  desc "Create an account with a preset password"
-  task :create_with_password, [ "account", "password" ] => [ "environment" ] do |t,args|
+  desc "Validate Password Strength"
+  task :validate_password_strength, ["password"] => [ "environment" ] do |t,args|
```

Comment:
i am not strongly against it but is is there a better place for this task than `account`? Do you see a strong connection to the account?

Reply:
Created password.rake and moved it there